pcsx2/gui/AppCoreThread.cpp

```diff
@@ -427,21 +427,23 @@ static void _ApplySettings( const Pcsx2Config& src, Pcsx2Config& fixup )
 		// the BIos isn't loaded until after initial calls to ApplySettings.
 
 		gameName = L"Booting PS2 BIOS... ";
-	}
 
 	//Till the end of this function, entry CRC will be 00000000
-	if (!gameCRC.Length())
+	CDVD_SourceType cdvdsrc(g_Conf->CdvdSource);
+	if ((!gameCRC.Length()) && ((cdvdsrc != CDVD_SourceType::NoDisc)))
 	{
 		Console.WriteLn(Color_Gray, "Patches: No CRC found, using 00000000 instead.");
 		gameCRC = L"00000000";
 	}
 
 	// regular cheat patches
-	if (fixup.EnableCheats)
+	if ((fixup.EnableCheats) && (cdvdsrc != CDVD_SourceType::NoDisc))
+	{
 		gameCheats.Printf(L" [%d Cheats]", LoadPatchesFromDir(gameCRC, GetCheatsFolder(), L"Cheats"));
+	}
 
-	// wide screen patches
-	if (fixup.EnableWideScreenPatches)
+		// wide screen patches
+	if ((fixup.EnableWideScreenPatches) && (cdvdsrc != CDVD_SourceType::NoDisc))
```

Comment:
You can just use one set of brackets: `if (fixup.EnableWideScreenPatches && cdvdsrc != CDVD_SourceType::NoDisc)`
The same goes for the other lines.

And there's an unnecessary tab before the comment above this line.